CGCharter-1727386911.html

```diff
@@ -62,28 +62,59 @@ <h2 id="goals">
         Incubate new proposals which build on or complement the Social Web WG recommendations.
       </li>
     </ul>
-  
+
     <h2 id="scope-of-work">
       Scope of Work
     </h2>
-    <p class="remove">
-      {TBD: Describe topics that are in scope. For specifications that the CLA
-      patent section applies to, it is helpful to describe the scope in a way
-      that it is clear what types of technologies will be defined in
-      specifications, as opposed to adoption by reference or underlying
-      technology not defined in the proposed spec. Key use cases are often
-      helpful in describing scope. If no specifications will be defined in the
-      group that the CLA patent section applies to, the charter should clearly
-      state that. A clear scope is particularly important where patent
-      licensing obligations may apply.}
+    <p>
+      The group will maintain errata and editor&apos;s drafts for the following recommendations from the Social Web Working Group:
+    </p>
+    <ul>
+      <li>ActivityPub</li>
+      <li>Activity Streams 2.0 Core</li>
+      <li>Activity Vocabulary</li>
+      <li>Webmention</li>
+      <li>Linked Data Notifications</li>
+      <li>Micropub</li>
+      <li>WebSub</li>
+    </ul>
+    <p>
+      It will maintain errata and editor&apos;s drafts for the following Notes from the Social Web Working Group:
+    </p>
+    <ul>
+      <li>JF2</li>
+      <li>Post Type Discovery</li>
+      <li>Social Web Protocols</li>
+      <li>IndieAuth</li>
+    </ul>
+    <p>
+      The group will maintain the Activity Streams 2.0 JSON-LD context document and add new extension terms as they become widely implemented.
+    </p>
+    <p>
+      The group will maintain the Activity Streams 2.0 OWL ontology.
+    </p>
+    <p>
+      The group will develop extensions to ActivityPub and Activity Streams 2.0, and other specifications as needed, to address new use cases and requirements.
+    </p>
+    <p>
+      The group will develop and document integration between ActivityPub and other protocols in a social web stack, such as Webfinger, HTML, and HTTP Signature.
```

Comment:
```suggestion
      The group will develop and document integration between ActivityPub and other protocols, such as Webfinger, HTML, and HTTP Signature.
```

I don't think the charter should take a position on which protocols are in a colloquial 'social web stack', and if it does, I think it's confusing to refer to WebMention that way since it is not mentioned anywhere in AS2/AP.

Hopefully this suggestion is agreeable, as it leaves WebMention work in scope while taking 'defining which protocols constitute the social web stack' as out of charter scope.